app/models/project.rb

```diff
@@ -9,7 +9,7 @@ class Project < ApplicationRecord
   include BitbucketProject
 
   HAS_DEPENDENCIES = false
-  STATUSES = ['Active', 'Deprecated', 'Unmaintained', 'Help Wanted', 'Removed']
+  STATUSES = ['Active', 'Deprecated', 'Unmaintained', 'Help Wanted', 'Removed', 'Hidden'].freeze
```

Comment:
'Hidden' is used on many projects in the database -- IMO we should document it here (and then I fix up some logic for this below)